rllib/examples/fractional_gpus.py

```diff
@@ -74,26 +75,33 @@
     # - num_gpus=0.4 + num_gpus_per_worker=0.1 (2 workers) -> 0.6
     #   -> 1 tune trial should run in parallel.
 
-    config = {
+    config = (
+        AlgorithmConfig()
         # Setup the test env as one that requires a GPU, iff
         # num_gpus_per_worker > 0.
-        "env": GPURequiringEnv if args.num_gpus_per_worker > 0.0 else "CartPole-v0",
-        # How many GPUs does the local worker (driver) need? For most algos,
-        # this is where the learning updates happen.
-        # Set this to > 1 for multi-GPU learning.
-        "num_gpus": args.num_gpus,
+        .environment(
+            GPURequiringEnv if args.num_gpus_per_worker > 0.0 else "CartPole-v0"
```

Comment:
can we upgrade to `v1`? if I'm not mistaken, this doesn't exist anymore in recent `gym` releases`

Reply:
Done in a separate PR. Feel like this shouldn't be in here. CartPole-v1 might indeed behave slightly differently, so we go t to be careful not to break any tuned examples.

Reply:
I think it just has another reward scale so we need to adjust test that depend on it.